core/blocks.js

```diff
@@ -12,14 +12,15 @@
 
 /**
  * A mapping of block type names to block prototype objects.
- * @name Blockly.Blocks
+ * @namespace Blockly.blocks
```

Comment:
Not quite sure what we want to do here - I think the alias should possibly be Blockly.Blocks? @cpcallen